### PR TITLE
Update tile style to contain actions

### PR DESCRIPTION
**Pull request by cbourgois**
Update tile styles to contain actions.
Remove tile's mixins.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3140,6 +3141,18 @@ <h4 class="oui-header_4 oui-tile__title">List definition</h4>
                                         <small>(quux)</small>
                                     </dd>
                                 </dl>
+                                <div class="oui-tile__actions">
+                                    <div class="oui-dropdown">
+                                        <button class="oui-action-menu-button oui-dropdown__trigger"
```
Comment (antleblanc) [inline index.html]: +`type="button"`

File: index.html [hunk 2]
```diff
@@ -3140,6 +3141,18 @@ <h4 class="oui-header_4 oui-tile__title">List definition</h4>
...
+                                            aria-haspopup="true" aria-expanded="true" aria-label="Server: actions">
+                                            <svg xmlns="http://www.w3.org/2000/svg" viewBox="-1 -1 74 74">
```
Comment (antleblanc) [inline index.html]: Why these `viewBox` values? (precisely `-1`)?

File: src/oui/oui-tile/tile.less [hunk 3]
```diff
@@ -1,7 +1,116 @@
...
+    display: flex;
+
+    border-top: 1px solid @oui-tile-item-border-color;
+    padding: @oui-tile-item-top-spacing 0;
+
+    &:last-child {
+      border-bottom: 1px solid @oui-tile-item-border-color;
+    }
+
+    // stylelint-disable max-nesting-depth
```
Comment (antleblanc) [inline src/oui/oui-tile/tile.less]: There is no `.stylelintrc` file nor scripts in `package.json` which is running stylelint, so we can get rid of this line I guess.

File: src/oui/oui-tile/tile.less [hunk 4]
```diff
@@ -1,7 +1,116 @@
...
+    &_button {
+      padding: 0;
+
```
Comment (antleblanc) [inline src/oui/oui-tile/tile.less]: Useless empty line.

File: src/oui/oui-tile/tile.less [hunk 5]
```diff
@@ -1,7 +1,116 @@
...
+    color: @oui-tile-definition-description-color;
+    font-size: @oui-tile-definition-description-font-size;
+    font-weight: @oui-font-bold;
+    margin-left: 0;
+
+    &_vertical-sub {
+      vertical-align: sub;
+    }
+
+    small {
```
Comment (antleblanc) [inline src/oui/oui-tile/tile.less]: Would you prefer using a class instead?

File: index.html (hunk 1 above)
Comment (cbourgois) [reply index.html]: Done ;)

**Pull request merged**